website/static/css/users.css

```diff
@@ -0,0 +1,52 @@
+.mainContainer .showcaseSection .logos {
```

Comment:
does it need to be nested like this?

Reply:
I can remove mainContainer but I think we should keep `.showcaseSection .logos` so that styles don't effect other pages.